### ISSUE TITLE
Some dependencies should be devDependencies

### ISSUE THREAD

**Issue by sr258** [leolabs/json-autotranslate#74]
I have json-autotranslate in the devDependencies of my project and when I install my project I get a deprecation warning from tslint. That's why I've checked where it's from and it's a dependency of json-autotranslate. The dependencies referenced in package.json could be cleaned up so that the dependencies only include those that are really required on runtime (so that they aren't added to other projects on install).

From the looks of it, you might be able to move these dependencies to devDependencies:

-    @types/node
-    dotenv
-    ts-node
-    tslint
-    typescript

**Comment by leolabs** [comment]
Good catch! This should be a quick fix, I'll try to get it done this weekend.
